lib/l10n/app_de.arb

```diff
@@ -330,6 +330,54 @@
   "pageChatNoChatMessage": "Du kannst mit deiner*m Fahrer*in chatten, sobald deine Fahrt angenommen wird.",
   "pageChatMessageBarHint": "Schreibe eine Nachricht",
 
+  "pageHomeTitle": "Home",
+  "pageHomeSearchButton": "Mitfahrt suchen",
+  "pageHomeCreateButton": "Fahrt erstellen",
+
+  "rideEventPendingTitle": "Mitfahrt angefragt!",
+  "rideEventApprovedTitle": "Mitfahrt bestätigt!",
+  "rideEventRejectedTitle": "Mitfahrt abgelehnt!",
+  "rideEventWithdrawnTitle": "Mitfahrt verworfen!",
+  "rideEventCancelledByDriverTitle": "Fahrt abgesagt!",
+  "rideEventCancelledByRiderTitle": "Mitfahrt abgesagt!",
+  "rideEventPendingMessage": "{username} hat eine Mitfahrt angefragt",
+  "@rideEventPendingMessage": {
```

Comment:
in the german arb file, you don't need these placeholders. They are taken from the english file